plugins/backup/veeam/src/main/java/org/apache/cloudstack/backup/VeeamBackupProvider.java

```diff
@@ -57,7 +57,7 @@
     public static final String BACKUP_IDENTIFIER = "-CSBKP-";
 
     public ConfigKey<String> VeeamUrl = new ConfigKey<>("Advanced", String.class,
-            "backup.plugin.veeam.url", "http://localhost:9399/api/",
+            "backup.plugin.veeam.url", "https://localhost:9398/api/",
```

Comment:
@DaanHoogland I've changed the default to use https, as per their release notes http is deprecated in future. Ref https://helpcenter.veeam.com/docs/backup/rest/whatsnew_14.html?ver=95u4 cc @PaulAngus @borisstoyanov